### ISSUE TITLE
Adding / specifying spacing between each bit group (total col width) and to the last column..

### ISSUE THREAD

**Issue by dry-75** [facelessuser/HexViewer#73]
Hello,
I'm not sure this is an issue or missing feature, may be it is possible to do this but..

How can I specify the empty spacing between the bit group columns, and from last bit group on the line to the interpreted (sometimes strings & printables) last column on the line?

For example, the spacing between the first column with byte addresses and first column data is different to the distance between each data column. I want to be able to control this , configure.  Ideally able to control all this white space / empty display distance. 
This can really help reading data.

Also, it seems not possible to configure that last column display /view - the ASCII part (if I'm not mistaken this is same as last column of output from hexdump -C  utility  (canonical hex+ASCII display).
Could be good to be able to configure that part too, like add custom coloring for the recognized strings...

**Comment by dry-75** [comment]
Poong ....  I guess this project is dead .. ?

**Comment by facelessuser** [comment]
Nope, project isn't dead, I just support a lot of things, and sometimes miss notifications of issues. FWIW, this project already does all the things I intend it to do, so I haven't added **new** features in quite a bit.

Spacing itself is not configurable. This is also not calling any external utility to create the hex dump.

I assume you are bothered my the fact that the address and ascii columns have two spaces? If so, this was done just to make the regex easier to detect where byte groups start and end. I guess we could tweak it to use just one space as we ended up adding colons as well.

If you are talking about byte grouping, that is controlled by existing settings that are documented (how many bytes are grouped without spaces).

If your issue is the first case, I can reduce those most likely to single space, but that Westworld be low priority as it is cosmetic.

**Comment by dry-75** [comment]
Apologies , for writing long message and I guess not being clear.

No, I want to be able to control / set the empty space between each byte(s) column. This is to see it clearly. 
And preferably also be able to set/control space between address column and the last/right-most columns (with ASCII stuff in).

So it seems you saying, this is not controllable ... Pity.

**Comment by facelessuser** [comment]
Not currently. Maybe a visual example can make your request more clear? Can you show what exactly you hope to configure the output to look like?

**Comment by dry-75** [comment]
Ok,
But it's really just be able to control space between groups, e.g. :

aaaa<->bbbb    vs   e.g  aaaa<---->bbbb

**Comment by facelessuser** [comment]
@gir-bot add T: feature, P: maybe

**Comment by dry-75** [comment]
> @gir-bot add T: feature, P: maybe

I don't know what the eyes reaction is or suppose to be. 
But if it 's still un-clear : want to have clear spacing between group bytes, so I don't have to tune my glasses.

Is this still vague .. ?

**Comment by facelessuser** [comment]
> I don't know what the eyes reaction is or suppose to be.

That's the bot reacting which signifies it saw that I was issuing it a command.

> Is this still vague .. ?

I understand it. I'm just not sure yet if it will be implemented or not. I'm open to it, but I haven't gotten around to doing anything with it yet.

**Comment by dry-75** [comment]
Ah ok, thank you for attention.
